donjuan/cell.py

```diff
@@ -62,32 +69,30 @@ def y(self) -> int:
 
     @property
     def n_sides(self) -> int:
-        return len(self.faces)
+        return type(self)._n_sides
```

Comment:
```suggestion
        return self._n_sides
```

I think?

Reply:
Oh, I see it's a class variable. Never seen this done before, but makes sense I guess

Reply:
It's gross and I hate it lol. But I needed to be able to reference the number of sides of a `Cell` from the `Grid`.

Reply:
Without instantiating a cell, I mean.